js/classes/entities/midground/weapon/sword.js

```diff
@@ -49,6 +57,16 @@ class Sword {
         return 7;
     };
 
+    upgrade() {
+        if (this.damage === Sword.LEVEL_1_DAMAGE) {
```

Comment:
Are you adding the upgrade system we had talked about originally?

Reply:
You bet!